test/library/test_mm.py

```diff
@@ -1,7 +1,13 @@
+from contextlib import nullcontext
+
 import pytest
 import torch
 from helpers import random_tensor
 
+from quanto.library import disable_extensions
+from quanto.tensor.core import group, ungroup
+from quanto.tensor.packed import pack_weights, unpack_weights
```

Comment:
Agree we should not have two unpack methods. Here you could use torch.ops.quanto.unpack instead.

Reply:
Done !